### PR TITLE
Memory leak fix and some minor changes

### PR DESCRIPTION
**Pull request by lneir**
code checked in from Sadda/GS, please review.

### CODE REVIEW

File: minuet/symphony/Paragon.Runtime/PackagedApplication/ParagonPackageVerifier.cs [hunk 1]
```diff
@@ -6,6 +7,8 @@
 
 namespace Paragon.Runtime.PackagedApplication
 {
+    //Excluding from code coverage Temporarily; can include once Digital Signing is implemented
+    [ExcludeFromCodeCoverage]
```
Comment (jonmclachlan) [inline minuet/symphony/Paragon.Runtime/PackagedApplication/ParagonPackageVerifier.cs:11]: I have horrid memory.  Would it be possible to get a jira ticket to remind us to revert this temporary change once we get signing up and working?

Also, why does static analysis happiness depend on code signing?

File: minuet/symphony/Paragon.Runtime/App.config [hunk 2]
```diff
@@ -10,15 +10,6 @@
             <setting name="MaxRolledLogFiles" serializeAs="String">
                 <value>5</value>
             </setting>
```
Comment (lneir) [inline minuet/symphony/Paragon.Runtime/App.config:12]: Code here is out of date, please rebase from latest master then submit a new PR.

File: minuet/symphony/Paragon.Runtime/Plugins/RenderSideMessageRouter.cs [hunk 3]
```diff
@@ -176,8 +176,12 @@ public void InitializePlugins(CefListValue browserPlugins)
             functionInvokeMessage.MessageType = PluginMessageType.FunctionInvoke;
             functionInvokeMessage.Data = parameters != null ? parameters.ToString() : string.Empty;
 
-            // Add the call info into the pending calls for the browser
-            AddRemoteCallback(functionInvokeMessage, methodDescriptor.HasCallbackParameter ? null : callback);
+            // Add the call info into the pending calls for the browser if the return type
+            // is not void or the method has a callback arg.
+            if (!methodDescriptor.IsVoid || methodDescriptor.HasCallbackParameter)
+            {
+                AddRemoteCallback(functionInvokeMessage, methodDescriptor.HasCallbackParameter ? null : callback);
```
Comment (lneir) [inline minuet/symphony/Paragon.Runtime/Plugins/RenderSideMessageRouter.cs:183]: Couldn't 2nd arg in this method call be simplified with the added condition?

File: minuet/symphony/Paragon/App.xaml.cs [hunk 4]
```diff
@@ -46,7 +46,7 @@ protected override void OnStartup(StartupEventArgs e)
                 if (!_suppressSplashScreen)
```
Comment (lneir) [inline minuet/symphony/Paragon/App.xaml.cs:46]: ditto... need to rebase from master as this code has changed, otherwise merge conflict will occur.

**Pull request closed without merging**